src/esm_runscripts/filedicts.py

```diff
@@ -120,6 +147,9 @@ def __init__(self, full_config: dict, attrs_address: dict):
         self.component = component = attrs_address.split(".")[0]
         self.path_in_computer = pathlib.Path(self["path_in_computer"])
 
+        # possible paths for files:
+        location_keys = ["computer", "exp_tree", "run_tree", "work"]
```

Comment:
Would it make sense to move this below the `# Initialize the locations and complete paths`? In this way the whole initialization is together and clearer.